crypto/threads_pthread.c

```diff
@@ -57,10 +57,13 @@ CRYPTO_RWLOCK *CRYPTO_THREAD_lock_new(void)
      * We don't use recursive mutexes, but try to catch errors if we do.
      */
     pthread_mutexattr_init(&attr);
-#  if !defined(NDEBUG) && !defined(OPENSSL_NO_MUTEX_ERRORCHECK)
+#  if !defined (__TANDEM) && !defined (_SPT_MODEL_)
+#   if !defined(NDEBUG) && !defined(OPENSSL_NO_MUTEX_ERRORCHECK)
     pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_ERRORCHECK);
-# else
```

Comment:
Ouch, this looks wrong. This else has to be kept (indented by one space).

Reply:
WT... Where did it go? It was there before I did the `rebase`. Trying again.

Reply:
What it looks like in my branch:
https://github.com/rsbeckerca/openssl/blob/d453475b2c5456b7631587299955d69a02c36b99/crypto/threads_pthread.c#L60-L67
which puts the `#   else` in the correct place although it looks a little awkward given the code contained.

Reply:
This is of course wrong - there is missing `#   else` between the pthread calls.

Reply:
> 
> 
> This is of course wrong - there is missing `# else` between the pthread calls.

Ack. Sorry.

Reply:
Should be fixed now.